src/Uploader/Storage/GaufretteOrphanageStorage.php

```diff
@@ -59,7 +60,7 @@ public function __construct(StorageInterface $storage, SessionInterface $session
      */
     public function upload($file, string $name, string $path = null)
     {
-        if (!$this->session->isStarted()) {
+        if (!$this->session || !$this->session->isStarted()) {
```

Comment:
Checking the instance of the session variable should fix the PHPStan issue(s). 

```suggestion
        if (!$this->session instanceof SessionInterface || !$this->session->isStarted()) {
```

Reply:
Fixed in d7da202.